reverse-proxy.md

```diff
@@ -56,7 +56,7 @@ nextcloud/all-in-one:latest
 ---
 
 > [!IMPORTANT] 
-> If you need HTTPS between Nextcloud and the reverse proxy because it is running on a different server in the same network, simply add another reverse proxy to the chain that runs on the same server like AIO and takes care of HTTPS proxying (most likely via self-signed certificates). Another option would be to create a VPN between the server that runs AIO and the server that runs the reverse proxy which takes care of encrypting the connection.
+> In reverse proxy mode your reverse proxy needs to talks to AIO using HTTP, so if you need HTTPS between Nextcloud and the reverse proxy because it is running on a different server in the same network, simply add another reverse proxy to the chain that runs on the same server like AIO and takes care of HTTPS proxying (most likely via self-signed certificates). Another option would be to create a VPN between the server that runs AIO and the server that runs the reverse proxy which takes care of encrypting the connection.
```

Comment:
Thanks Zoey but I am not sure if this is the right place to mention HTTP usage?

Reply:
Also, I guess this needs some adjustements to be understandable for people that do not know the project yet...

Reply:
> Thanks Zoey but I am not sure if this is the right place to mention HTTP usage?

not sure, but I thought it is the best place to mention it without creating a new section in the guide

Reply:
Are you fine if I close this PR?

Reply:
if you want